src/main/scad/yavuz-demirhan/basket-burr/basketBurr.scad

```diff
@@ -0,0 +1,57 @@
+include <puzzlecad.scad>
+
+// Global settings
+$burr_scale = 10;
+$plate_width = 100;
+$burr_inset = 0.05;
+$joint_inset = 0.08;
+$burr_bevel = 1.0;
+$unit_beveled = false;
+
+*basket();
+*holder();
+*pieces();
+
+module basket() {
+    render(convexity = 2)
+    burr_piece([
+        "xxxxxx|xxxxxx|x....x|......|......|......",
+        "xxxxxx|xxxx{connect=fy+}xx|x....x|......|......|......",
+        "xxxxxx|......|......|......|......|......",
+        "xxxxxx|......|......|......|......|......",
+        "xxxxxx|xxx{connect=fy+}xxx|x....x|......|......|......",
+        "xxxxxx|xxxxxx|x....x|......|......|......"
+    ]); 
+}
+
+module holder() {
+    burr_plate([[
+        "......|......|......|......|......|......",
+        "......|......|...x{connect=my-}..|...x..|..xx..|..xx..",
+        "......|......|......|......|......|..x{connect=fz+}x{connect=mz+}.."
+    ], [
+        "......|......|......|......|......|......",
+        "......|......|...x{connect=my-}..|...x..|..xx..|..xx..",
+        "......|......|......|......|......|..x{connect=fz+}x{connect=mz+}.."
+    ]]);
+}
+
+module pieces() {
+    burr_plate([
+        ["xxxxxx|xxxxxx", "x....x|x....x"],
+        ["xxxxxx|xxxxxx", "x....x|x....x"],
+        ["xxxxxx|xxxxxx", "x....x|x....x"],
+        ["xx..xx|xxxxxx", "x....x|x....x"]
+    ]);
+}
+
+module holder_single() {
+    burr_plate([[
+        "......|......|......|......|......|......",
+        "......|......|...x{connect=my-}..|...x..|..xx..|..xx..",
+        "......|......|......|......|......|..xx..",
+        "......|......|......|......|......|..xx..",
+        "......|......|..x{connect=my-}...|..x...|..xx..|..xx..",
+        "......|......|......|......|......|......"]
+    ]);  
```

Comment:
By the way, there is a trick that will give you the handle as a single piece with the desired appearance. Try this:

    burr_plate([[
        "aabb|aabb", "a..b|a..b", "a...|...b", "a{connect=mz+}...|...b{connect=mz+}"
    ]]);

It will render as a single piece, but with beveling applied separately to the "a" and "b" subpieces.

Reply:
I thought about it a bit more, and in this case I think your approach is better... because the handle pieces will be laid out on the print bed identically to all the other pieces, for a more consistent look. But, that trick can be useful sometimes :)